roles/tpa_single_node/templates/manifests/guac/graphql/Deployment.yaml.j2

```diff
@@ -43,29 +42,48 @@ spec:
           command:
             - /opt/guac/guacgql 
           args:
-            - --gql-backend=ent
+            - --gql-listen-port={{ tpa_single_node_guac_graphql_port }}
+            - --gql-backend=ent 
             - --db-address=postgres://{{ tpa_single_node_pg_user }}:{{ tpa_single_node_pg_user_passwd }}@{{ tpa_single_node_pg_host }}:{{ tpa_single_node_pg_port }}/{{ tpa_single_node_pg_db }}?sslmode={{ tpa_single_node_pg_ssl_mode }}
             - --db-driver=postgres
-          workingDir: /opt/guac
+            - --db-migrate=false
```

Comment:
The options gql-listen-port, db-debug and gql-debug would be removed at the final version?

Reply:
@mrrajan, 

Certainly for the debug options

I don't think so regarding the listen port at least if we want to keep control of the port number. 
That's actually the main cause we lost access to the graphql service. It's not clear the default port is 8080 and the logs are not advertising it.